midpoint/install/system-requirements/index.adoc

```diff
@@ -190,135 +196,133 @@ This is important to keep in mind when choosing the right HA strategy.
 You need access to the infrastructure where these servers are running, as well as to the source and target systems.
-In most cases, the infrastructure is prepared by the administrators on the customer site.
+
+Refer to the xref:/midpoint/release/[midPoint Releases] documentation for software requirements.
 
 === Basic Single-Node Deployment
 
-The following schema represents a basic scenario:
+The following schema represents a basic deployment environment:
 
-//image::environment-schema-basic.png[]
-image::midpoint-environment-schema-basic.svg[title="Basic single-node midPoint deployment"]
+.Basic single-node midPoint deployment
+image::midpoint-environment-schema-basic.svg["Basic midPoint deployment schema with midPoint and its repository in the center; lines showing access routes to example source and target resources and user's computer. Optional VPN is placed between midPoint and the computer"]
 
 In the center of the schema, the largest rectangle represents a virtual machine, usually running Linux, with the xref:/midpoint/install/bare-installation/distribution/#purpose-and-quality[basic required set of tools] installed.
+MidPoint runs on this virtual machine.
+The database repository preferably runs on a separate virtual machine, albeit possibly on the same physical server.
 
 ==== Shared Database Repository
 
-For the database repository (DB), an existing DB server is usually used.
-You have access to the DB using the SQL via the TCP/IP from the midPoint server. 
-Do not forget to configure the firewalls to enable communication over the used ports—5432 typically for PostgreSQL, 1433 for MS SQL, 1521 for Oracle, etc.
+For the database repository (DB), you can use an existing DB server or set up a new dedicated one.
+The midPoint server needs to have access to it using SQL via TCP/IP.
+Do not forget to configure the firewalls to enable communication over the used ports.
 
 The database needs to be configured prior to midPoint installation so that the person who installs midPoint can configure the midPoint instance correctly.
-In the single-node mode, the DB can be placed on a separate DB server.
+If you deploy in containers, see xref:/midpoint/install/containers/#deployment-schema[] for details.
 
 ==== Notification System
 
 If e-mail notifications are needed, access to the SMTP server and an account with send privileges is required.
 If you need SMS notifications, you also need access to an SMS gateway and have the account privileges to send SMS.
 
-==== Secure Remote Deployment
+==== Secure Access to Your Deployment
+
+If your setup requires access to your midPoint deployment over the public Internet, you need to secure the communication between end users' computers and the midPoint instance.
+One of the options is to use a virtual private network (VPN) for everyone who needs to access midPoint.
+VPN provides a tunnel from the user's machine to midPoint (or the load balancer before it).
```

Comment:
Yes, but for some people having simple HTTPS is good enough. This seems it's necessary to have something heavyweight like VPN.